sites/kit.svelte.dev/package.json

```diff
@@ -14,23 +14,24 @@
 		"@sveltejs/adapter-vercel": "workspace:^",
 		"@sveltejs/amp": "workspace:^",
 		"@sveltejs/kit": "workspace:^",
-		"@sveltejs/site-kit": "6.0.0-next.36",
-		"@types/d3-geo": "^3.0.3",
-		"@types/node": "^20.4.9",
+		"@sveltejs/site-kit": "6.0.0-next.42",
+		"@types/d3-geo": "^3.0.4",
+		"@types/node": "^20.6.0",
 		"browserslist": "^4.21.10",
 		"flexsearch": "^0.7.31",
-		"lightningcss": "^1.21.5",
-		"magic-string": "^0.30.2",
+		"lightningcss": "^1.21.8",
+		"magic-string": "^0.30.3",
 		"marked": "^9.0.0",
+		"prettier": "^3.0.3",
```

Comment:
i'm surprised this would be added when there's no script calling it. I think it could be removed for now